### PR TITLE
Replace Travis-CI with GitHub actions

### PR DESCRIPTION
**Pull request by felddy**
Replace Travis-CI with GitHub actions.  

BadColorfulLhasaapso-size_restricted

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: actions/checkout@v1
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```
Comment (jsf9k) [inline .github/workflows/build.yml:21]: Heh, by coincidence I just had to use this bit of python-fu elsewhere.  Small world.

**Review by jsf9k — APPROVED** [review]
The king is dead.  Long live the king!

**Review by dav3r — APPROVED** [review]
SEND IT!  ☃️ 

⚠️ Reminder for all of us to not pull this into [skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) until https://github.com/ansible/ansible-lint/issues/590 is resolved.

**Comment by jsf9k** [comment]
Is the gif from Erik the Viking?

**Comment by felddy** [comment]
> Is the gif from Erik the Viking?

I think so.  I was looking for "funny viking funeral"

**Pull request merged**